homeassistant/components/homeassistant/scene.py

```diff
@@ -26,6 +26,36 @@
 from homeassistant.helpers.state import HASS_DOMAIN, async_reproduce_state
 from homeassistant.components.scene import DOMAIN as SCENE_DOMAIN, STATES, Scene
 
+
+def _convert_states(states):
+    """Convert state definitions to State objects."""
+    result = {}
+
+    for entity_id in states:
+        cv.entity_id(entity_id)
```

Comment:
Should we overwrite the entity_id with the returned value from the validator? That's usually how we use validators.